pages/common/pandoc.md

```diff
@@ -1,7 +1,15 @@
 # pandoc
 
-> General markup converter.
+> Convert documents between various formats.
 
 - Convert file to pdf (the output format is automatically determined from the output file's extension):
 
 `pandoc {{input.md}} -o {{output.pdf}}`
+
+- Convert a file to a specific output format (useful for when the extension alone is ambiguous):
```

Comment:
Maybe we should expand `-t` to `--to` because its not very clear what it stands for. Initially I thought it stood for type.